flag/flags.go

```diff
@@ -0,0 +1,12 @@
+package flag
+
+// Flags provides the ability to create and communicate feature flags.
+type Flags uint64
+
+// Flags is a bitset
+const (
+	Debug Flags = 1 << iota
+	SamplingSet
```

Comment:
"For example, in all carriers (even http), debug is read. In others
they are not"

I meant to say

"For example, in all carriers (even http), debug is read. In http,
only the debug flag is expected."